docs/user/editor/routes.md

```diff
@@ -15,37 +15,43 @@ Clicking the 🔍 button (in the top toolbar) with a route selected adjusts the
 
 Some of the fields to be filled in are required before you can ‘Save and Continue’ and move to the next step, these are:
 
-- **Short name** – name of the service/route, this may just be a number
-- **Long name** – the full name of the route, often including the origin and destination
-- **Route Type** – the type of vehicle/transport used on the route
-Other fields in this section are optional and do not have to be filled in, these are:
-- **Description** – a description of the route, do not simply repeat the information in ‘Long name’
-- **URL** – a link to a webpage with information on the specific route, such as the timetable
-- **Color** – if a route has a color (for use in trip planners etc) it can be assigned here
-- **Text color** – if a route has a text color (for use in trip planners etc) it can be assigned here
-- **Comments** – any additional information about the route can be placed here
-
-Once all the required fields, and any of the desired optional fields, are filled in click ‘Save and continue’.
-
-
-## Review
-
-This final page allows you to show if a route has been completely entered, and if it has whether it has been checked and approved for inclusion in the GTFS feed.
+- **Status** - takes the following values: 
+    - **In-Progress** - showing a route has not been completely entered.
+    - **Pending Approval** - a route has all the information entered and is awaiting a senior person to sign it off
+    - **Approved** - all the above stages have been completed.
+- **Publicly Visible?** - this must be set to "Yes" for the route to be included in a GTFS output. 
+- **route_id** - an identifier for the route. A randomly generated ID is provided by default.
+- **route_short_name** – name of the service/route, this may just be a number
+- **route_long_name** – the full name of the route, often including the origin and destination
+- **route_type** – the type of vehicle/transport used on the route
 
-### Status
-
-There are three options for **Status**:
-
-- **In-Progress** - showing a route has not been completely entered.
-- **Pending Approval** - a route has all the information entered and is awaiting a senior person to sign it off
-- **Approved** - all the above stages have been completed.
+Other fields in this section are optional and do not have to be filled in, these are:
```

Comment:
This list is not rendering as bullets for some reason:

![image](https://user-images.githubusercontent.com/56846598/88347851-c9b23f80-cd19-11ea-9ce7-c26b81f1bbff.png)

Reply:
I can't seem to replicate this issue. Here's how this section appears to me: 
![image](https://user-images.githubusercontent.com/63798641/89664049-8f61aa00-d8a4-11ea-943d-af1bf5ac48e0.png)

Reply:
I know. In `routes.md`, insert a blank line before the first `agency_id` bullet. Perhaps `mkdocs` is a bit stricter than github in interpreting the markdown list.

Reply:
I also just noticed that after each field, there is a mix of `-` and `–` dashes? Pick one kind throughout.